### PR TITLE
Create ASR Job History.workbook

### PR DESCRIPTION
**Pull request by tmalhotra2428**
## PR Checklist

* [x] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.
* [x] Validate your changes using one or more of the [testing](../Documentation/Testing.md) methods.

### If adding or updating templates:
* [x] post a screenshot of templates and/or gallery changes
* [x] ensure your template has a corresponding gallery entry in the gallery folder
* [x] If you are adding a new template, add your team and template/gallery file(s) to the CODEOWNERS file. CODEOWNERS entries should be teams, not individuals
* [x] ensure all steps have meaningful names
* [x] ensure all parameters and grid columns have display names set so they can be localized
* [x] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [x] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [x] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [x] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

File: Workbooks/Azure Backup/ASR ReplicatedItems /ASR ReplicatedItems.workbook [hunk 1]
```diff
@@ -0,0 +1,1220 @@
...
+      "conditionalVisibility": {
+        "parameterName": "Workspaces",
+        "comparison": "isNotEqualTo"
+      },
+      "name": "Heading2"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "This report helps you view information on your replicated items over a specified duration of time. [Learn more](https://go.microsoft.com/fwlink/?linkid=2211463&clcid=0x409)"
```
Comment (naba6327108) [inline Workbooks/Azure Backup/ASR ReplicatedItems /ASR ReplicatedItems.workbook:143]: Please change to - This report provides information on the replicated items over a specified duration of time.

File: Workbooks/Azure Backup/ASR ReplicatedItems /ASR ReplicatedItems.workbook [hunk 2]
```diff
@@ -0,0 +1,1220 @@
...
+              "durationMs": 5184000000
+            }
+          },
+          {
+            "id": "efede5fa-f577-4766-b9b6-6ba4e525f842",
+            "version": "KqlParameterItem/1.0",
+            "name": "VaultSubscription",
+            "label": "Vault Subscription",
+            "type": 6,
+            "description": "Use to filter for vaults within a specific subscription",
```
Comment (naba6327108) [inline Workbooks/Azure Backup/ASR ReplicatedItems /ASR ReplicatedItems.workbook:331]: Change to - Filter for vaults within a specific subscription

File: Workbooks/Azure Backup/ASR ReplicatedItems /ASR ReplicatedItems.workbook [hunk 3]
```diff
@@ -0,0 +1,1220 @@
...
+              "value::all"
+            ]
+          },
+          {
+            "id": "256c7e33-df90-4956-aaf3-699aeaad912g",
+            "version": "KqlParameterItem/1.0",
+            "name": "VaultResourceGroup",
+            "label": "Vault Resource Group",
+            "type": 2,
+            "description": "Use to filter for vaults within a specific resource group",
```
Comment (naba6327108) [inline Workbooks/Azure Backup/ASR ReplicatedItems /ASR ReplicatedItems.workbook:359]: Change to - Filter for vaults within a specific resource group

File: Workbooks/Azure Backup/ASR ReplicatedItems /ASR ReplicatedItems.workbook [hunk 4]
```diff
@@ -0,0 +1,1220 @@
...
+              "value::all"
+            ]
+          },
+          {
+            "id": "849a6401-cbaf-44b9-a733-0819f8923795",
+            "version": "KqlParameterItem/1.0",
+            "name": "SearchItemHybrid",
+            "label": "Search Item",
+            "type": 1,
+            "description": "Use to search for an item by name",
```
Comment (naba6327108) [inline Workbooks/Azure Backup/ASR ReplicatedItems /ASR ReplicatedItems.workbook:387]: Change to - Search for an item by name

File: Workbooks/Azure Backup/ASR ReplicatedItems /ASR ReplicatedItems.workbook [hunk 5]
```diff
@@ -0,0 +1,1220 @@
...
+    },
+    {
+      "type": 11,
+      "content": {
+        "version": "LinkItem/1.0",
+        "style": "paragraph",
+        "links": [
+          {
+            "id": "bac30f99-eaf9-4c7f-8705-cc263b8e1e88",
+            "cellValue": "https://aka.ms/BCDRAuditReportTemplates",
```
Comment (naba6327108) [inline Workbooks/Azure Backup/ASR ReplicatedItems /ASR ReplicatedItems.workbook:19]: Please change link to - https://learn.microsoft.com/en-us/azure/backup/azure-policy-configure-diagnostics

**Review by naba6327108 — CHANGES_REQUESTED** [review]
Please make the following changes:
1. I have added some comments for text and url change. Please make that change. 
2. Also kindly structure the replication health errors bag. 
3. And, please make the replicated item a hyperlink

File: Workbooks/Azure Backup/ASR Job History/ASR Job History.workbook [hunk 6]
```diff
@@ -0,0 +1,1247 @@
...
+      "conditionalVisibility": {
+        "parameterName": "Workspaces",
+        "comparison": "isNotEqualTo"
+      },
+      "name": "Heading2"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "This report helps you view information on your successful and failed ASR jobs over a specified duration of time. Currently only jobs triggered on replicated items and recovery plans are shown in this report. [Learn more](https://go.microsoft.com/fwlink/?linkid=2211463&clcid=0x409)"
```
Comment (naba6327108) [inline Workbooks/Azure Backup/ASR Job History/ASR Job History.workbook:143]: Please change to - This report provides information on the successful and failed site recovery jobs over a specified duration of time. Currently only jobs triggered on replicated items and recovery plans are shown in this report.

File: Workbooks/Azure Backup/ASR Job History/ASR Job History.workbook [hunk 7]
```diff
@@ -0,0 +1,1247 @@
...
+              "value::all"
+            ]
+          },
+          {
+            "id": "256c7e33-df90-4956-aaf3-699aeaad912f",
+            "version": "KqlParameterItem/1.0",
+            "name": "VaultResourceGroup",
+            "label": "Vault Resource Group",
+            "type": 2,
+            "description": "Use to filter for datasources within a specific resource group",
```
Comment (naba6327108) [inline Workbooks/Azure Backup/ASR Job History/ASR Job History.workbook:415]: Change to - Filter for datasources within a specific resource group

File: Workbooks/Azure Backup/ASR Job History/ASR Job History.workbook [hunk 8]
```diff
@@ -0,0 +1,1247 @@
...
+              "durationMs": 7776000000
+            }
+          },
+          {
+            "id": "efede5fa-f577-4766-b9b6-6ba4e525f844",
+            "version": "KqlParameterItem/1.0",
+            "name": "VaultSubscription",
+            "label": "Vault Subscription",
+            "type": 6,
+            "description": "Use to filter for datasources within a specific subscription",
```
Comment (naba6327108) [inline Workbooks/Azure Backup/ASR Job History/ASR Job History.workbook:387]: Change to - Filter for datasources within a specific subscription

**Review by naba6327108 — CHANGES_REQUESTED** [review]
Please make the reqd changes:
1. Changes suggested in the comments
2. Please make the order of columns in the grid as : replicated item, vault, subscription, location, jobs operation, status, st time, duration

**Comment by naba6327108** [comment]
We will take these changes adhoc. Approving for a first level experience.

**Comment by tmalhotra2428** [comment]
> Please make the reqd changes:
> 
> 1. Changes suggested in the comments
> 2. Please make the order of columns in the grid as : replicated item, vault, subscription, location, jobs operation, status, st time, duration

**Review by naba6327108 — APPROVED** [review]

**Pull request merged**